Sources/ChatUI/ChatInChannel/MessageField/Giphy/View/GiphyAttributionMarkView.swift

```diff
@@ -0,0 +1,32 @@
+import SwiftUI
+
+class CustomTabBarController: UITabBarController {
+    var tabBarHeight: CGFloat {
+        tabBar.frame.size.height
+    }
+}
+
+struct GiphyAttributionMarkView: View {
+    
+    private let PADDING_ABOVE_TAB = 2.5
```

Comment:
```suggestion
    private let paddingAboveTab = 2.5
```
please modify snake case to lower camel case :)